bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -402,7 +402,7 @@ spec:
         - mountPath: /etc/openvswitch
           name: etc-openvswitch
         - mountPath: /etc/ovn/
-          name: etc-openvswitch
+          name: var-ovn-etc
```

Comment:
Not sure I'm following this 2nd commit. Why are you trying to mount DBs inside ovnkube-node? We would only want to check metrics for the DBs on the masters, and they should already have access to the DBs.

Reply:
I should have explained it. Its because of this metric: https://github.com/ovn-org/ovn-kubernetes/blob/8a315ee7109780544fcc4ad7d3afbfb62e621bcd/go-controller/pkg/metrics/ovn_db.go#L255
Its hard coded to `/etc/ovn/`.
LMK if this is a problem and I can rework upstream.

Reply:
I removed the mounting of OVN DBs. This will cause an error log message to occur in OVN-K node: https://github.com/ovn-org/ovn-kubernetes/blob/8a315ee7109780544fcc4ad7d3afbfb62e621bcd/go-controller/pkg/metrics/ovn_db.go#L258

I will fix suppress this with a PR upstream.

Reply:
Thanks. I dont think we should be querying DB stats from the node if possible.